services/core/shared/pouchdb.js

```diff
@@ -30,27 +30,47 @@ const createDb = async (name, idxList = []) => {
 	logger.debug(`Creating/opening database ${name}...`);
 	const db = new PouchDB(name, { auto_compaction: true });
 
-	idxList.forEach(async idxName => {
-		logger.debug(`Setting up index ${idxName}...`);
+	// const availableIndexes = [];
+	// (await db.getIndexes())
+	// 	.indexes.forEach(index => availableIndexes.push(index.name));
+	// console.log(availableIndexes);
+
+	idxList.forEach(async propName => {
+		let idxName = 'idx-'.concat(db.name.split('/')[1]).concat('-');
+		if (typeof propName === 'string') {
+			idxName = idxName.concat(propName);
+			propName = [propName];
+		} else {
+			idxName = idxName.concat(propName.join('-'));
+		}
+
+		logger.debug(`Setting up index ${name}/${idxName.split('-').slice(2).join('-')}...`);
+
 		await db.createIndex({
 			index: {
-				fields: [idxName],
+				fields: propName,
+				name: idxName,
 			},
 		});
 	});
 
 	return db;
 };
 
-const getDbInstance = async (collectionName) => {
+const dbLogger = {};
+
+const getDbInstance = async (collectionName, idxList = []) => {
+	if (!dbLogger[collectionName]) dbLogger[collectionName] = Logger(`pouchdb-${collectionName}`);
+	// const cLogger = dbLogger[collectionName];
```

Comment:
Kindly consider using the `cLogger`. Refer https://github.com/LiskHQ/lisk-service/pull/146.